app/models/concerns/host_params.rb

```diff
@@ -37,7 +37,7 @@ def host_params
       return cached_host_params unless cached_host_params.blank?
       hp = host_inherited_params
       # and now read host parameters, override if required
-      host_parameters.each {|p| hp.update Hash[p.name => p.value] }
+      host_parameters.authorized(:view_params).each {|p| hp.update Hash[p.name => p.value] }
```

Comment:
nitpick: why not `host_parameters.authorized(:view_params).each {|p| hp[p.name] = p.value }`?